src/static/riot/submissions/submission_management.tag

```diff
@@ -0,0 +1,266 @@
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+            </tr>
+            <tr>
+                <td class="collapsing">
+                    <div class="ui checkbox">
+                        <input type="checkbox"> <label></label>
+                    </div>
+                </td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+            </tr>
+            <tr>
+                <td class="collapsing">
+                    <div class="ui checkbox">
+                        <input type="checkbox"> <label></label>
+                    </div>
+                </td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+            </tr>
+            <tr>
+                <td class="collapsing">
+                    <div class="ui checkbox">
+                        <input type="checkbox"> <label></label>
+                    </div>
+                </td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+            </tr>
+            <tr>
+                <td class="collapsing">
+                    <div class="ui checkbox">
+                        <input type="checkbox"> <label></label>
+                    </div>
+                </td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+                <td></td>
+            </tr>
+            </tbody>
+            <tfoot class="full-width">
+            <tr>
+                <th colspan="11">
+                    <div class="ui right floated pagination menu">
+                        <a class="icon item"><i class="left chevron icon"></i></a>
+                        <a class="item">1</a>
+                        <a class="item">2</a>
+                        <a class="item">3</a>
+                        <a class="item">4</a>
+                        <a class="icon item"><i class="right chevron icon"></i></a>
+                    </div>
+                </th>
+            </tr>
+            </tfoot>
+        </table>
+    </div>
+    <!-- Javascript -->
+    <script>
+        $(document).ready(function () {
+            $('.ui.sticky')
+                .sticky({
+                    context: '.ui.compact.celled.table',
+                    offset: 75
+                })
+        })
+    </script>
+    <!-- CSS -->
```

Comment:
same thing here with comment, prob good to just do 2 spaces

Reply:
removed comment